forms-web-app/src/config.js

```diff
@@ -4,11 +4,13 @@ module.exports = {
     url: process.env.APPEALS_SERVICE_API_URL,
   },
   fileUpload: {
-    abortOnLimit: process.env.FILE_UPLOAD_ABORT_ON_HITTING_FILE_SIZE_LIMIT === 'true',
```

Comment:
express-fileupload has a built in way to handle file uploads that are too big.

This works - but we do not have much control over exactly how. The issue being that express-fileupload throws a 413 if the file size is too big. All good. However, we need to display that error on the page that had the issue, which the middleware made impossible (that I could see, anyway). 

Instead, we are going to rely on our own internal validator for file size (which works) - but has the draw back of accepting the full file first, and then determining file size. I'm not a huge fan of this, as I think this is a regression over what we have. But given the time constraints, this does meet the AC still.

Reply:
Yeah, I agree with this as a first iteration. I agree that uploading the file and handling it is not great, but much better than it sending a 413 error with no control.

We can put a ticket in the backlog to investigate doing this better (personally, I doubt we'll ever get to it but so long as it's captured)